csrankings-2.csv

```diff
@@ -1260,6 +1260,7 @@ Eugenia Ternovska,Simon Fraser University,http://www.cs.sfu.ca/~ter,NOSCHOLARPAG
 Eugenia Ternovskaia,Simon Fraser University,http://www.cs.sfu.ca/~ter,NOSCHOLARPAGE
 Eugenio Zimeo,University of Sannio,https://www.unisannio.it/en/users/Eugenio-Zimeo,o8-JBUEAAAAJ
 Eugénie Foustoucos,AUEB,https://uregister-ldap.aueb.gr/en/content/applied-mathematics-informatics-finance-and-operational-research-sector,t1TycUYAAAAJ
+Eui-Nam Huh,Kyung Hee University,http://icnslab.net/#/introduction

```

Comment:
Missing Google Scholar ID. Also, this commit, like all of your commits, is not appropriately descriptive. Please rename them.